tarsier/tag_utils.ts

```diff
@@ -183,11 +183,23 @@ window.tagifyWebpage = (tagLeafTexts = false) => {
       continue;
     }
 
-    const idStr = isTextInsertable(el) ? `{${idNum}}` : `[${idNum}]`;
+    let idStr: string;
+    if (isInteractable(el)) {
+      if (isTextInsertable(el))
+        idStr = `{${idNum}}`;
+      else
+        idStr = `[${idNum}]`;
+    } else {
+      idStr = `#${idNum}`;
```

Comment:
do we still want some parenthesis here? phone numbers for example could be mistaken for tags with this

Reply:
resolved